dev/README.md

```diff
@@ -0,0 +1,32 @@
+# Setting up a Development Environment
+This describes how to set up a development environment suitable for working on the automation for `gatling-puppet-load-test`
+
+## What you get
+The beaker scripts in the `beaker` directory will set up a machine with a jenkins server, a copy of this repo (which contains the gatling executables), `sbt` for running gatling, and an installation of `jenkins-job-builder` configured to point at the jenkins server.
+
+## Requirements
+This guide assumes you have a centos-6 machine/VM with:
+  * ssh key authentication for the root user (for `beaker`)
+  * `puppet` installed and in the `PATH`
+
+This will be your jenkins/gatling box 
+
+To actually run gatling against a SUT, you'll need to set up another machine with some flavor of puppetserver/PE depending on what you're trying to test. Running tests isn't covered in this guide though.
+
+It's also important to note that if you want to work on the cobbler provisioning portion of the automation, it's much more involved. There will be a separate guide for that at some point.
+
+## Setting up the jenkins/gatling box
+To run the beaker script, you must first edit `beaker/target_machine.yml` and replace `target_hostname` with the hostname or IP of your system.
+
+`cd` to the root of the repo, and run.
+```
+beaker --log-level debug --hosts dev/beaker/target_machine.yml --tests dev/beaker/
+```
+
+If everything goes well, the beaker output should show no errors.
+
+Jenkins should be available on port `8080` of your machine.
```

Comment:
Ah, I see what's wrong.  I was running my beaker command from inside the `dev` directory (because I created a `Gemfile` there to deal with installing beaker for me).  The `apply_manifests.rb` has a line in it where it does a glob on `./dev/manifests/*.pp`, which was basically a no-op for me since that path didn't exist.

It'd probably be better to list out the manifests directly in an array literal, so that it will fail if any are not found.  I wouldn't expect those to change often enough to where globbing them will save us a ton of effort.  (You can create a separate ticket for that, though.)

Also if you do put relative paths into the code I think you should probably have devs run their commands with `./dev` as their current working directory (because that will be the most sensible place to put a Gemfile), so paths should be relative from there.